src/components/Swap/Summary/index.tsx

```diff
@@ -27,7 +27,6 @@ enum ReviewState {
   REVIEWING,
   ALLOWING,
   ALLOWANCE_FAILED,
-  TRADE_CHANGED,
```

Comment:
I combined REVIEWING & TRADE_CHANGED since clicking swap in both of these states has the same flow now